src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -185,12 +183,6 @@ namespace VariableSpeedCoils {
             }
         }
 
-        if (present(OnOffAirFlowRat)) {
```

Comment:
This is just setting up a default value that's a constant, so use a default value instead and eliminate the `Optional`

Reply:
Awesome.